modules/services/timed.nix

```diff
@@ -0,0 +1,45 @@
+{
+  config,
+  lib,
+  ...
+}:
+
+let
+  cfg = config.services.timed;
+
+  onOff = cond: if cond then "on" else "off";
+in
+{
+  meta.maintainers = [
+    (lib.maintainers.stepbrobd or "stepbrobd")
+  ];
+
+  options.services.timed = {
+    enable = lib.mkOption {
+      default = true;
+      type = lib.types.bool;
+      description = ''
+        Enables the timed NTP client daemon.
+      '';
+    };
+
+    servers = lib.mkOption {
+      default = config.networking.timeServers;
+      defaultText = lib.literalExpression "config.networking.timeServers";
+      type = lib.types.listOf lib.types.str;
+      description = ''
+        The set of NTP servers from which to synchronise.
+      '';
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
```

Comment:
This block will never be executed if `enable = false`, so it does nothing if a user want to disable timed.